hiss/application/admin.py

```diff
@@ -111,11 +153,14 @@ class ApplicationAdmin(admin.ModelAdmin):
             "num_hackathons_attended",
             custom_titled_filter("number of attended hackathons"),
         ),
+        date_range_filter,
         ("datetime_submitted", custom_titled_filter("date submitted")),
     )
+
     list_display = (
         "first_name",
         "last_name",
+        "user_email",
```

Comment:
I'm not entirely sure if this will work right out of the box, but try this first:
```suggestion
        "user__email",
```

If it doesn't work, do this:

```python
class ApplicationAdmin(admin.ModelAdmin):
    # ....
    list_display = (
        # ...
        "user_email",
        # ...
    )

    # ...
    def user_email(self, obj: Application) -> str:
        return obj.user.email
    # ...
```
Adding the user's email as a field to the model adds duplication to the database, which should be avoided whenever possible for two reasons:
1. It takes more space up, which costs more money when we use a managed instance.
2. It invites bugs. I can easily imagine myself forgetting to set the `user_email` field on the `Application` model, and that can lead to unexpected behavior down the line.